e2e/portalicious/pages/RegistrationsPage.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, Locator } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+
+class RegistrationsPage extends BasePage {
+  readonly page: Page;
+  readonly tableRows: Locator;
+  readonly selectAllPasCheckbox: Locator;
```

Comment:
```suggestion
  readonly selectAllRegistrationsCheckbox: Locator;
```

Let's use the new terminology as much as possible

Reply:
Done!